databuilder/databuilder/extractor/oracle_metadata_extractor.py

```diff
@@ -0,0 +1,45 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import (  # noqa: F401
+    Any, Dict, Iterator, Union,
+)
+
+from pyhocon import ConfigFactory, ConfigTree  # noqa: F401
+
+from databuilder.extractor.base_oracle_metadata_extractor import BaseOracleMetadataExtractor
+
+
+class OracleMetadataExtractor(BaseOracleMetadataExtractor):
```

Comment:
Why is there a `BaseOracleMetadataExtractor` as well as an `OracleMetadataExtractor`? This approach makes sense for generic SqlAlchemy extractor which can theoretically be extended to any type of relational database as well as for Postgres which can serve both Postgres and Redshift. Are there multiple implementations of Oracle that are slightly different which may require separate implementations?

Reply:
You're right! I merged the two classes into one.